api/XMLHttpRequest.json

```diff
@@ -1112,9 +1112,15 @@
             "edge_mobile": {
               "version_added": true
             },
-            "firefox": {
-              "version_added": true
-            },
+            "firefox": [
+              {
+                "version_added": true
+              },
+              {
+                "version_added": "30",
+                "notes": "Starting in Firefox 30, synchronous requests on the main thread have been deprecated due to their negative impact on performance and the user experience. Therefore, the <code>async</code> parameter may not be <code>false</code> except in a <code>Worker</code>."
+              }
+            ],
             "firefox_android": {
               "version_added": true
```

Comment:
`firefox_android` should be the same as `firefox`.

Reply:
Like @ExE-Boss said, this also applies to Firefox for Android.